Formula/gobo.rb

```diff
@@ -0,0 +1,94 @@
+class Gobo < Formula
+  desc "Free and portable Eiffel tools and libraries"
+  homepage "http://www.gobosoft.com/"
+  url "https://downloads.sourceforge.net/project/gobo-eiffel/gobo-eiffel/20.01/gobo2001-src.tar.gz"
```

Comment:
What's the license for this?

Reply:
MIT:  http://www.gobosoft.com/eiffel/gobo/license.html